packages/carbon-graphs/tests/unit/controls/Graph/GraphResize-spec.js

```diff
@@ -586,4 +586,27 @@ describe('Graph - Resize', () => {
       expect(graph.config.canvasHeight).toBe(250);
     });
   });
+  describe("Only when graph container is present", () => {
```

Comment:
Can we add another test for `when the graph container is not present`?

Reply:
When the graph container is not present, we are restricting from executing the statements which set the canvas width, scale the graph and translate the graph. A unit test to check if those statements are not getting executed was not possible, thats the reason I wrote an unit test to check if the statements are being executed if the graph container is actually present.